openpype/tools/launcher/widgets.py

```diff
@@ -181,6 +191,46 @@ def _start_animation(self, index):
             self._animated_items.add(action_id)
             self._animation_timer.start()
 
+    def on_context_menu(self, point):
+        """Creates menu to force skip opening last workfile."""
+        index = self.view.indexAt(point)
+        if not index.isValid():
+            return
+
+        action_item = index.data(ACTION_ROLE)
+        if not self.model.is_application_action(action_item):
+            return
+
+        menu = QtWidgets.QMenu(self.view)
+        checkbox = QtWidgets.QCheckBox("Force not open last workfile",
```

Comment:
I'd just change the text a little bit, otherwise not problem 
```suggestion
        checkbox = QtWidgets.QCheckBox("Skip opening last workfile.",
```